src/frontend/src/utils/countdown.ts

```diff
@@ -100,3 +101,80 @@ const calculateTimeRemaining = (
       .padStart(2, "0"),
   ];
 };
+
+// Prettify the number into a string: "mm:ss"
+function prettifySeconds(a: number): string {
```

Comment:
- can it become more than 99 minutes and would the function and display handle this?

- side note: is there an API like Intl.NumberFormat would format this out of the box?

Reply:
Currently it cannot be more than 99minutes; I just copied the existing code but I'll have a look at how we can format it better